edgedb/protocol/protocol.pyx

```diff
@@ -80,14 +80,22 @@ cdef class QueryCodecsCache:
     def __init__(self, *, cache_size=1000):
         self.queries = LRUMapping(maxsize=cache_size)
 
-    cdef get(self, str query, IoFormat io_format):
-        return self.queries.get((query, io_format), None)
+    cdef get(
+        self, str query, IoFormat io_format,
```

Comment:
fwiw, I prefer the `name: type` style for consistency with Python source.

Reply:
Somehow it didn't work for me here (I really tried). But I'll be refactoring this code anyways.